specification/batch/resource-manager/Microsoft.Batch/stable/2021-06-01/BatchManagement.json

```diff
@@ -1,7 +1,8 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "BatchManagement",
+    "title": "BatchManagementClient",
+    "description": "The Batch Management Client.",
```

Comment:
please reference this : https://github.com/Azure/sdk-release-request/issues/1800#issuecomment-888745462

Reply:
Added the title to the readme (also found a similar inconsistency with the data plane and fixed it)